client/web/web.go

```diff
@@ -164,16 +165,23 @@ func NewServer(opts ServerOpts) (s *Server, err error) {
 		opts.LocalClient = &tailscale.LocalClient{}
 	}
 	s = &Server{
-		mode:        opts.Mode,
-		logf:        opts.Logf,
-		devMode:     envknob.Bool("TS_DEBUG_WEB_CLIENT_DEV"),
-		lc:          opts.LocalClient,
-		cgiMode:     opts.CGIMode,
+		mode:    opts.Mode,
+		logf:    opts.Logf,
+		devMode: envknob.Bool("TS_DEBUG_WEB_CLIENT_DEV"),
+		lc:      opts.LocalClient,
+		cgiMode: opts.CGIMode,
+		// In enforcePrefix, we add the necessary leading '/'. If we did not
```

Comment:
Let's go ahead and just move the comment down to the `if opts.Prefix` block below, since that's where we actually implementing the slash stripping and all.  And no need to set `pathPrefix` here, since the alternative is just that it's an empty string anyway.

Reply:
Done, thanks!